src/Component/api_listing.latte

```diff
@@ -1,5 +1,13 @@
 <div class="api-listing">
     <div n:foreach="$apis as $version => $versionApis" class="listing-verions listing-version-{$version}">
+        {php $showSummary = false}
```

Comment:
preco nie {var $showSummary = } ?

Reply:
neviem tak som si zvykol v nette 2.4 ked nieco zdeprecatovali, ze uz davam vsade to {php}